pages/projects/[id]/index.tsx

```diff
@@ -2,6 +2,9 @@ import { type GetServerSidePropsContext } from "next";
 import ProjectLayout from "@/layouts/Project";
 import { getServerSideSession } from "@/utils/session";
 import { Project } from "@prisma/client";
+import { HiOutlineCommandLine } from "react-icons/hi2";
```

Comment:
Lets replace these icons with lucide.dev's. `main` does not have these icon libraries anymore.

Reply:
Sure.. Will do that.. Thanks for the feedback